tests/test_simulation.py

```diff
@@ -764,6 +756,35 @@ def test_eff_noise(matrices):
         )
 
 
+@pytest.mark.parametrize(
+    "noise_sample,",
+    [
+        ("depolarizing"),
+        ("dephasing"),
+    ],
+)
+def test_three_level_noise(noise_sample, matrices):
+    np.random.seed(123)
+    reg = Register3D.from_coordinates([(0, 0, 0), (0, 0, 1)], prefix="q")
+    seq = Sequence(reg, MockDevice)
+    seq.declare_channel("ryd", "rydberg_local", "q0")
+    seq.declare_channel("raman", "raman_local", "q1")
+    duration = 2500
+    pulse = Pulse.ConstantPulse(duration, np.pi, 0, 0)
+    seq.add(pulse, "ryd")
+    seq.add(pulse, "raman")
+    sim = QutipEmulator(
+        sampler.sample(seq),
+        reg,
+        MockDevice,
+        sampling_rate=0.01,
+        config=SimConfig(
+            noise=noise_sample,
+        ),
+    )
+    assert all([op.shape == (9, 9) for op in sim._collapse_ops])
```

Comment:
This is a fine thing to check but we can (and should) go much further. Must of the tests done with qubits should be extensible to qutrits. 
It's important to run simple simulations and assert that the results match our expectations.